specification/botservice/resource-manager/Microsoft.BotService/stable/2017-12-01/botservice.json

```diff
@@ -30,10 +30,10 @@
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.BotService/botServices/{resourceName}": {
       "put": {
         "tags": [
-          "BotService"
+          "Bot"
         ],
         "description": "Creates a Bot Service. Bot Service is a resource group wide resource type.",
-        "operationId": "BotServices_Create",
+        "operationId": "Bots_Create",

```

Comment:
Changing the Operation Group that is associated with an Operation is a breaking change from the code gen perspective.  For instance with the Go generator, each operation group has a client, and operations are dangled off of it.

Because this spec is in the `stable` folder, we don't accept breaking changes inside of an API Version without escalation.

Reply:
@marstr This is a new API, so no breaking changes since it has never been published. @olydis merged it the state from azure-rest-api-specs-pr last week as a starting point and he created the "stable" folder.

Reply:
Ah got it, let's move everything to a `preview` folder then. I'll notify all the SDK owners to make sure they aren't caught off-guard.

Reply:
Got it, moving to a preview folder in the next batch of changes.